src/test/pmempool_rm/TEST10

```diff
@@ -0,0 +1,100 @@
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+#     * Redistributions of source code must retain the above copyright
+#       notice, this list of conditions and the following disclaimer.
+#
+#     * Redistributions in binary form must reproduce the above copyright
+#       notice, this list of conditions and the following disclaimer in
+#       the documentation and/or other materials provided with the
+#       distribution.
+#
+#     * Neither the name of the copyright holder nor the names of its
+#       contributors may be used to endorse or promote products derived
+#       from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+# A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+# OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+#
+# pmempool_rm/TEST10 -- test for pmempool rm in interactive mode
+#
+
+. ../unittest/unittest.sh
+
+require_test_type medium
+
+setup
+
+LOG=out${UNITTEST_NUM}.log
+rm -f $LOG && touch $LOG
+
+# Create pmemlog, pmemblk and pmemobj pools
+expect_normal_exit $PMEMPOOL$EXESUFFIX create log $DIR/pool.log
+expect_normal_exit $PMEMPOOL$EXESUFFIX create blk 512 $DIR/pool.blk
+expect_normal_exit $PMEMPOOL$EXESUFFIX create obj $DIR/pool.obj
+
+# Create poolset with replica
+create_poolset $DIR/pool.set 32M:$DIR/pool.part1:z 32M:$DIR/pool.part2:z
+
+# Create pmemobj pools from the poolset
+expect_normal_exit $PMEMPOOL$EXESUFFIX create obj $DIR/pool.set
+
+check_files\
+	$DIR/pool.log\
+	$DIR/pool.blk\
+	$DIR/pool.obj\
+	$DIR/pool.set\
+	$DIR/pool.part1\
+	$DIR/pool.part2
+
+
+chmod 400\
+	$DIR/pool.log\
+	$DIR/pool.blk\
+	$DIR/pool.obj\
+	$DIR/pool.set\
+	$DIR/pool.part1\
+	$DIR/pool.part2
+
+echo y | expect_normal_exit $PMEMPOOL$EXESUFFIX rm $DIR/pool.log
+echo N | expect_normal_exit $PMEMPOOL$EXESUFFIX rm $DIR/pool.blk
+echo Y | expect_normal_exit $PMEMPOOL$EXESUFFIX rm $DIR/pool.obj
+echo n | expect_normal_exit $PMEMPOOL$EXESUFFIX rm $DIR/pool.part1
+echo n | expect_normal_exit $PMEMPOOL$EXESUFFIX rm $DIR/pool.part2
+
+check_files\
+	$DIR/pool.blk\
+	$DIR/pool.part2\
+	$DIR/pool.set\
+	$DIR/pool.part1
+
+check_no_files\
+	$DIR/pool.log\
+	$DIR/rep.obj
+
+
+expect_normal_exit $PMEMPOOL$EXESUFFIX rm -s $DIR/pool.set 2>&1 <<EOF
```

Comment:
this test is supposed to verify that we ask if we should really remove the file/poolset. But I don't think heredoc syntax actually causes a failure on not fully consumed stream. Which means that this will succeed even if `pmempool rm` does not require any input.